dev/io.openliberty.microprofile.health.internal.common/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -19,4 +19,7 @@ mpHealth=MicroProfile Health
 mpHealth.desc=Configuration for MicroProfile Health.
 
 fileUpdateInterval=Interval of time at which to update the file-based health checks.
-fileUpdateInterval.desc=The value that you specify sets the interval at which the health checks are queried for the file-based health checks. The value is a number followed by an optional time unit of ms for milliseconds or s for seconds. If no time unit is specified, the value is in seconds by default.
\ No newline at end of file
+fileUpdateInterval.desc=The value that you specify sets the interval at which the health checks are queried for the file-based health checks. The value is a number followed by an optional time unit of ms for milliseconds or s for seconds. If no time unit is specified, the value is in seconds by default.
+
+fileCreateInterval=Interval of time at which to create the file-based health checks.
+fileCreateInterval.desc=The value that you specify sets the interval at which the health checks are queried until an UP status is resolved and the health check file is created. The value is a number followed by an optional time unit of ms for milliseconds or s for seconds. If no time unit is specified, the value is in seconds by default. The default value is 100ms.
```

Comment:
"If no time unit is specified, the value is in seconds by default. The default value is 100ms." change to "If no time unit is specified for a value, the value is in seconds by default. If no value is specified, the default value is 100 ms."